packages/lib-classifier/src/components/Classifier/Classifier.js

```diff
@@ -41,6 +41,12 @@ function Classifier({
     }
   }
 
+  /** This runs when a volunteer:
+      - views a new project
```

Comment:
I don't think it's possible to change the project without leaving the Classify page and unmounting the classifier.

Reply:
True!